src/components/CardItem.js

```diff
@@ -1,47 +1,47 @@
 import React from "react";
 import { makeStyles } from "@material-ui/core/styles";
 import Card from "@material-ui/core/Card";
-import { CardMedia, } from "@material-ui/core";
-import CardActions from "@material-ui/core/CardActions";
+import { CardMedia } from "@material-ui/core";
 import CardContent from "@material-ui/core/CardContent";
-import Button from "@material-ui/core/Button";
 import Typography from "@material-ui/core/Typography";
 
 const useStyles = makeStyles({
   root: {
     minWidth: 150,
-    margin:10
+    margin: 10,
   },
   bullet: {
     display: "inline-block",
     margin: "0 2px",
-    transform: "scale(0.8)"
+    transform: "scale(0.8)",
   },
   title: {
-    fontSize: 14
+    fontSize: 14,
   },
   pos: {
-    marginBottom: 12
-  }
+    marginBottom: 12,
+  },
 });
 
 export default function CardItem(props) {
   const classes = useStyles();
-
   return (
     <Card className={classes.root} variant="outlined">
-         <CardMedia
-          component="img"
-          alt="Contemplative Reptile"
-          height="240"
-          image={"http://images.tmdb.org/t/p/w300/"+props.path}
-          title="Contemplative Reptile"
-        />
+      <CardMedia
+        component="img"
+        alt="Contemplative Reptile"
+        height={300}
+        image={"http://images.tmdb.org/t/p/w300/" + props.path}
```

Comment:
`http://images.tmdb.org/t/p/w300/` make it const

Reply:
done